### PR TITLE
Add cinder beasts to base-hunting

### PR DESCRIPTION
**Pull request by mwest152**
These 2 rooms are mapped and accessable.  Rooms beyond these are part of
a maze and will require special attention to navigate.

### CODE REVIEW

File: data/base-hunting.yaml [hunk 1]
```diff
@@ -498,6 +498,10 @@ hunting_zones:
   - 7278
   - 7279
   - 7271
+  # https://elanthipedia.play.net/Maelshyvean_cinder_beast
```
Comment (rpherbig) [inline data/base-hunting.yaml]: Could you include the approximate range for this hunting zone?

**Comment by rpherbig** [comment]
There appears to be a merge conflict which needs to be resolved before I can merge this.

File: data/base-hunting.yaml (hunk 1 above)
Comment (mwest152) [inline data/base-hunting.yaml]: Ah yeah oops, 450-600

**Comment by mwest152** [comment]
I will fix it!

**Comment by mwest152** [comment]
I like this opportunity to try and learn to do stuff, but I'm kinda stumped as to what I did wrong.

**Comment by rcuhljr** [comment]
Two of you just changed the same spot in the file at the same time.

**Pull request merged**